server/contracts/GalaxyToken.sol

```diff
@@ -0,0 +1,28 @@
+//Contract based on [https://docs.openzeppelin.com/contracts/3.x/erc721](https://docs.openzeppelin.com/contracts/3.x/erc721)
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+
+contract GalaxyToken is ERC721URIStorage, Ownable {
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
+    address payable _owner;
+
+    constructor(address payable owner) ERC721("GalaxyToken", "NFT") {
+        _owner = owner;
+    }
+
+    function mintToken(address to) public payable virtual {
+        require(msg.value >= 10, "Not enough ETH sent; check price!");
```

Comment:
So right now `msg.value >= 10` means that all the **GalaxyTokens** costs 10 ETH?

Reply:
In my tests, this seems to be 0.00000001 ETH. So my guess this that is is Wei? Will need to look at the documentation a bit more.

Reply:
Aa, ofc it is in `Wei`. I was thinking it was in ETH and hence a bit expensive 😄